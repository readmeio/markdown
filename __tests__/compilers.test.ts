import { mdast, mdx } from '../index';

describe('ReadMe Flavored Blocks', () => {
  it('Embed', () => {
    const txt = '[Embedded meta links.](https://nyti.me/s/gzoa2xb2v3 "@embed")';
    const ast = mdast(txt);
    const out = mdx(ast);
    expect(out).toMatchSnapshot();
  });

  it.skip('Variables', () => {
    expect(mdx(mdast('<<variable:user>>'))).toMatchInlineSnapshot(`
      "<<variable:user>>
      "
    `);
  });

  it.skip('Glossary Items', () => {
    expect(mdx(mdast('<<glossary:owl>>'))).toMatchInlineSnapshot(`
      "<<glossary:owl>>
      "
    `);
  });

  it('Emojis', () => {
    expect(mdx(mdast(':smiley:'))).toMatchInlineSnapshot(`
      ":smiley:
      "
    `);
  });
});
