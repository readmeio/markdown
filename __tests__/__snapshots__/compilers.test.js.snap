// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ReadMe Flavored Blocks > Embed 1`] = `
"[Embedded meta links.](https://nyti.me/s/gzoa2xb2v3 "@nyt")
"
`;
